sdk.yaml

```
@@ -281,6 +281,11 @@ Realtime:
           (optional functionality).
   Connection:
     .specification: RTC2
+    Ping:
+      .specification: RTN13
+      .synopsis: |
+        Sends a message to the Ably service and asynchronously returns the time to response, or an error if the operation failed.
```

```suggestion
        Sends a heartbeat message to the Ably service and returns the time taken to receive a corresponding heartbeat in response, or an error if the operation failed.
```

I propose removing "asynchronously", since for example in ably-go or ably-rust, I'd likely make the API look synchronous (e.g. `func (c *Client) Ping() (*Pong, error)` or `fn ping() -> Result<Pong, Error>`)